.dotnet/api/OpenAI.netstandard2.0.cs

```diff
@@ -4,6 +4,8 @@ public class OpenAIClient {
         public OpenAIClient(ApiKeyCredential credential, OpenAIClientOptions options);
         public OpenAIClient(ApiKeyCredential credential);
         protected internal OpenAIClient(ClientPipeline pipeline, OpenAIClientOptions options);
+        public OpenAIClient(string apiKey, OpenAIClientOptions options);
+        public OpenAIClient(string apiKey);
```

Comment:
@annelo-msft : Did you originally add this temporarily to avoid having to fix all the code that was using the implicit cast of ApiKeyCredential that was removed, and we should remove it before merging this PR? Or did you intent to add this API?

Reply:
Both.  IIRC we had discussed with architects that we would do this when we had removed the implicit cast so that users wouldn't have to `new ApiKeyCredential` everywhere per usability.  But if you don't want to do that, or want to address in a separate PR, we can do it differently in this PR.